lib/vicadmin/validate.go

```diff
@@ -62,8 +62,9 @@ type Validator struct {
 }
 
 const (
-	GoodStatus = template.HTML(`<i class="icon-ok"></i>`)
-	BadStatus  = template.HTML(`<i class="icon-attention"></i>`)
+	GoodStatus     = template.HTML(`<i class="icon-ok"></i>`)
+	BadStatus      = template.HTML(`<i class="icon-attention"></i>`)
+	DefaultVCHName = `VCH`
```

Comment:
@hickeng We already logged the failure if we cannot get the VCH name from vsphere. It will look weird if we set the defaultVCHName to something like `VCH_defaultName` or something that indicates the failure.